tests/Modules/MotdModule.Tests/HttpServiceTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using EvoSC.Modules.Official.MotdModule.Models;
+using EvoSC.Modules.Official.MotdModule.Services;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace MotdModule.Tests;
+
+public class HttpServiceTests
+{
+    private const string TestServerUri = "https://directus.evoesports.de/items/motd?filter[server][_eq]=testserver";
+    private HttpService _service = new(new Logger<HttpService>(new LoggerFactory()));
+
+    [Theory]
+    [InlineData(TestServerUri)]
+    [InlineData("https://www.google.de")]
+    async Task GetAsyncTest(string uri)
+    {
+        var result = await _service.GetAsync(uri);
+        Assert.Equal(uri.Contains("evo") ? "This is a MOTD message served by the API. Including $f00styling." : "",
+            result);
+    }
+
+    [Fact]
+    async Task DataModelTest()
```

Comment:
I think we should mock the response instead here incase the above URL suddenly isn't available anymore. It also allows tests to run offline or behind a firewall.

Reply:
Hardcoded response. Checking functionality on GetAsync_Returns_Correct_Format_Test with a mock on the httpclient.